psamm/commands/fluxcoupling.py

```diff
@@ -43,7 +43,7 @@ def run(self):
 
         max_reaction = self._model.get_biomass_reaction()
         if max_reaction is None:
-            raise CommandError('The biomass reaction was not specified')
+            self.fail('The biomass reaction was not specified')
```

Comment:
this command is not trying to read the reaction from command line, but trying to get it from the model method, so it fails, it should just be a run time fail, right?